composer.json

```
@@ -13,19 +13,24 @@
         "bin-dir": "bin"
     },
     "require": {
-        "php": "^7.4",
-        "predis/predis": "^1.1.1",
+        "php": ">=8.0",
+        "predis/predis": "^1.1.8",
         "doctrine/cache": "~1.3",
-        "b1rdex/predis-compressible": "^0.5.0"
+        "b1rdex/predis-compressible": "^1.0",
+        "symfony/cache": "~5.0 || ~6.0",
+        "symfony/config": "~5.0 || ~6.0",
+        "symfony/dependency-injection": "~5.0 || ~6.0",
+        "symfony/http-foundation": "~5.0 || ~6.0",
+        "symfony/http-kernel": "~5.0 || ~6.0",
+        "symfony/yaml": "~5.0 || ~6.0"
     },
     "require-dev": {
-        "m6web/php-cs-fixer-config": "2.0.*",
-        "atoum/atoum": "^3.3.0",
-        "atoum/atoum-bundle": "^2.0",
+        "m6web/php-cs-fixer-config": "^2.0",
```

```suggestion
        "m6web/php-cs-fixer-config": "2.0.*",
```

(we don't want to use the `2.1.0` version in this project... at least not in this PR)